couchdb/README.md

```diff
@@ -86,6 +79,7 @@ and then install the chart while overriding the `createAdminSecret` setting:
 $ helm install \
   --name my-release \
   --version=3.6.1 \
+  --set adminHash=true/false \
```

Comment:
I think adminHash is the hash (`-pbkdf2-d4b887da...`) rather than `true/false`? If it's not already described, it would be useful to note / link to documentation around how to create that value from a password.

Reply:
Well... you are right but if you set your own secret with your own adminHash, the script admin-hash-copy in statefulset.yaml won't run (this behavior is inherited from the previous chart, I discovered this limitation and thought it would be good to put it somewhere in the doc!). So yes, you could put the adminHash to the real value but the whole purpose of using your own secret is to hide sensitive data from helm.

It's definitely not a best practice but I simply kept the default behavior from the previous chart...

Reply:
I think we should avoid overloading the meaning of `adminHash` value to support the bring-your-own secret use case. The [pattern used across Bitnami charts](https://github.com/helm/charts/blob/master/stable/keycloak/values.yaml#L62-L66) would be reasonable to copy and/or the proposal in #72 might support this.

Reply:
Sure...but that's another subject :) This PR is only and exclusively to simplify the adminHash secret. I've just improved the documentation so someone running into this would have a solution in the readme. If you merge this, we can definitely improve the chart with another PR and change the way existing secrets are declared.

Reply:
I'd be happy to help...but one step after the other. If we merge this and the autoSetup early this week, I ll be able to help for the existing secret PR